### ISSUE TITLE
Update labels: Issue Template: Mobile_responsiveness

### ISSUE THREAD

**Issue by chrismenke45** [hackforla/website#4483]
### Prerequisite
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started page](https://www.hackforla.org/getting-started) and attend an onboarding session.
2. Please make sure you have read our **[Hack for LA Contributing Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)** before you claim/start working on an issue.
### Overview
As developers, we need to add the labels from the [Missing series](https://github.com/hackforla/website/wiki/How-to-read-and-interpret-labels#missing-series) so that we can avoid the GitHub bot taking off the labels when devs add them to issues.
### Action Items
- [x] Navigate to `.github/ISSUE_TEMPLATE/Mobile_responsiveness.yml`
- [x] Replace the following line
   ```
   labels: ['role: design']
   ```
   with
   ```
   labels: ['role: design', 'Feature Missing', 'Complexity: Missing', 'size: missing']
   ```
- [x] Test the updated template using these [instructions](https://github.com/hackforla/website/issues/4483#issuecomment-1550766745)

**For Reviewers and Merge Team**
- To review this issue, follow the link below under "Link for Reviewers", then confirm the labels in the issue template match with those specified in the issue.

#### Link for Reviewers 
URL of the issue template on the test repository: https://github.com/rdhmdhl/website/issues/new?assignees=&labels=role%3A+design%2CFeature+Missing%2CComplexity%3A+Missing%2Csize%3A+missing&projects=&template=Mobile_responsiveness.yml&title=Mobile+Responsiveness+on+%5Bname+of+the+page%5D+%3A+%5Bquick+recap+of+the+bug+here%5D

**For PM, Merge Team, or Tech Lead**
- [x] Once the pull request associated with this issue is approved and merged, please update and edit epic #4307 by
  - [x] Checking off the dependency for this issue
  - [ ] If all dependencies are checked off, please move issue to the New Issue Approval column and remove the Dependency label

#### File and Code links you will need to work on this issue
- Directory to find the page in once you have it in your IDE: `.github/ISSUE_TEMPLATE/Mobile_responsiveness.yml`
- This issue is part of epic #4307

**Comment by Ahtaxam** [comment]
@chrismenke45 can i work on this issue?

**Comment by ExperimentsInHonesty** [comment]
@Ahtaxam Please choose an issue from the [prioritized backlog with the label good first issue](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22good+first+issue%22#column-7198257)

**Comment by roslynwythe** [comment]
Hi @rdhmdhl I added an additional Action Item with a link to instructions for testing.

**Comment by t-will-gillis** [comment]
Hey @rdhmdhl - before you send another PR, we noticed a typo in the code we gave you that will cause an error. Instead of the code in the description please use: 

`labels: ['role: design', 'Feature Missing', 'Complexity: Missing', 'size: missing']`

Our apologies- this code should be correct pending your testing. Thanks!

FYI I edited the description above to show the same changes...

**Comment by rdhmdhl** [comment]
Hi @roslynwythe and @t-will-gillis I have updated this file with the labels provided in this comment [above](https://github.com/hackforla/website/issues/4483#issuecomment-1550237935), and tried to test this with the [new instructions template](https://github.com/hackforla/website/issues/4307#issuecomment-1549888931). 

After changing the default repository to the repository that addresses this issue, selecting "Issues" in the Features settings, navigating to the issues tab, and creating a new issue using the "Mobile Responsiveness" template-- I do not see these new labels. Please see screenshots below.

Settings:
![Screen Shot 2023-05-16 at 8 51 49 PM](https://github.com/hackforla/website/assets/103082829/98b0df8c-d264-4cb0-b061-8797b9167573)

Labels in the Mobile responsiveness template:
![Screen Shot 2023-05-16 at 8 52 13 PM](https://github.com/hackforla/website/assets/103082829/536a42eb-25a0-4500-92c7-2ab1c5cc10ab)

However, when navigate to the code section of this fork, I see the labels are updated in the ["Mobile Responsiveness" template](https://github.com/rdhmdhl/website/blob/update-labels-4483/.github/ISSUE_TEMPLATE/Mobile_responsiveness.yml):
![Screen Shot 2023-05-16 at 8 53 34 PM](https://github.com/hackforla/website/assets/103082829/439f15b3-4709-45f5-be75-cdde33dcbbb6)

It seems like the issue templates/labels did not carry over to the new default branch. Would you like me to wait on making a new pull request until after this testing process can be verified?

**Comment by roslynwythe** [comment]
Hi @rdhmdhl you are almost there but just missing a step, in which you add the labels to your test repository.  I had to edit the instructions earlier today so I apologize if the instructions were not complete or were confusing.  Please see the step:  "Create labels corresponding to each label that was listed in the updated issue template. The color of the labels don't have to match those on the hackforla/website repository, but the text of the labels must match exactly."  The instructions also include a screenshot in case you need help finding the UI to create labels in the test repository.  As soon as you create the labels, you should be able to  bring up the issue template and see the labels.  Thank you for working with us to test the issue.   

For your convenience I've copied the instructions into a comment below.  Please feel free to send me a DM on Slack if you have any questions.

**Comment by roslynwythe** [comment]
### Instructions for testing updated issue templates:
- [x] Push your issue branch to your fork of the website repository, as described in [Section 3.1.a of CONTRIBUTING doc](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#31a-push-all-changes-to-your-issue-branch), for example:  `git push --set-upstream origin test-update-labels-1234`  
- [x] The response from the push command will include the URL for creating a pull request.  Copy/Paste the URL below for use after testing:  
[REPLACE THIS TEXT WITH URL FOR CREATING PR]
- [x] Browse to your fork of the website repository and click Settings 
<details><summary>Screenshot of Repository Settings</summary>

![image](https://github.com/hackforla/website/assets/5314153/8ecf2e55-2187-4ab9-acf1-2fd7548291a1)

</details> 

- [x] Change the default branch to the new issue branch

<details><summary>Screenshot of Default Branch Setting</summary>

![image](https://github.com/hackforla/website/assets/5314153/945fe02e-035f-4215-9c25-1d46c209d8c0)

</details> 

- [x] Further down on the settings page under Features, check the Issues checkbox

<details><summary>Screenshot of Issues Checkbox</summary>

![image](https://github.com/hackforla/website/assets/5314153/945fe02e-035f-4215-9c25-1d46c209d8c0)

</details> 

- [x] Browse to the Issues page, then click Labels

<details><summary>Screenshot - creating labels in the test Repo</summary>

![image](https://github.com/hackforla/website/assets/5314153/208e1b07-2478-4ed3-98a1-207de58d25c5)

</details> 

- [x] Create labels corresponding to each label that was listed in the updated issue template.  The color of the labels don't have to match those on the `hackforla/website` repository, but the text of the labels must match exactly.  
- [x] From the Issues page, click `New Issue` then locate the template updated by your issue, and click `Get Started` to open the issue template. 
- [x] Copy/Paste the URL of the issue template into the `Link for reviewers` section of the issue, replacing the text: [REPLACE THIS TEXT WITH URL OF THE ISSUE TEMPLATE ON THE TEST REPO]
 This will enable others to check the template.  
- [x] Verify that the labels listed in the issue all appear on the issue template.  If there is a discrepancy between the labels listed in the issue and the labels that appear on the issue template, ask a dev lead or merge team member for help.  
- [x] If the labels listed in the issue all appear on the issue template, complete the PR process by visiting the URL saved from the push command

**Comment by rdhmdhl** [comment]
Hi @roslynwythe thanks for clarifying here. Apologies for the back and forth. I've created the new labels now, should I remove the old ones or leave those?
 
 ![Screen Shot 2023-05-17 at 10 14 08 AM](https://github.com/hackforla/website/assets/103082829/8aae6914-8710-4907-a912-313c5899fe37)

**Comment by roslynwythe** [comment]
Hi @rdhmdhl Thank you for adding the labels; they are perfect.  There's no need to remove the existing labels.  I brought up the "Mobile_responsiveness" issue template in your repo and I copied the URL to that template into the `Link for Reviewers` section of the PR.  I checked the labels and they look great.  So you can check off those next two checkboxes and then proceed to the last one in which the PR is created.  Once you have a PR, please copy the "Link for Reviewers" section from the issue to the PR.  Thank you for your patience and let me know if you run into any problems.

**Comment by rdhmdhl** [comment]
Thanks @roslynwythe ! Pull request created with the link for reviewers.
